src/Core/src/PropertyMapper.cs

```diff
@@ -66,20 +92,27 @@ protected virtual void UpdatePropertyCore(string key, IElementHandler viewHandle
 
 		public void UpdateProperty(IElementHandler viewHandler, IElement? virtualView, string property)
 		{
-			if (virtualView == null)
+			if (virtualView == null || !viewHandler.CanInvokeMappers())
+			{
 				return;
+			}
 
-			UpdatePropertyCore(property, viewHandler, virtualView);
+			if (MergedMappers.TryGetValue(property, out var action))
+			{
+				action(viewHandler, virtualView);
+			}
```

Comment:
Is there a reason, perf or otherwise, why this no longer just calls UpdatePropertyCore? Seems the code is the same?

Reply:
The reason is that I'd like to get rid of that method (now not called anymore) in .NET10.
I moved everything here directly to have `!viewHandler.CanInvokeMappers()` being executed only once in `UpdateProperties`, but I cannot change the behavior of `UpdatePropertyCore` right now given it's a public API.